hadoop-ozone/interface-client/src/main/proto/OmClientProtocol.proto

```diff
@@ -307,7 +307,7 @@ message OMResponse {
   optional TenantAssignAdminResponse         TenantAssignAdminResponse     = 102;
   optional TenantRevokeAdminResponse         TenantRevokeAdminResponse     = 103;
 
-  optional GetS3VolumeResponse               getS3VolumeResponse           = 104;
+  optional GetS3VolumeContextResponse        getS3VolumeContextResponse    = 104;
```

Comment:
We should change the name of the request to match as well.

Reply:
done